app/models/key_pair.rb

```diff
@@ -23,11 +23,11 @@ def generate_keys
     private_key_path = dir_path + 'id_rsa'
     public_key_path = dir_path + 'id_rsa.pub'
 
-    unless system("ssh-keygen -f #{private_key_path} -P '' -t rsa")
+    unless system("ssh-keygen -f #{private_key_path} -P '' -t rsa -q")
```

Comment:
we could just use backtick here 
```
`ssh-keygen -f #{private_key_path} -P '' -t rsa -q`
```

Reply:
I'm actually checking the exit status of the command here, backticks would return the command's stdout output.

Reply:
got it 👍